docs/azure_arc_jumpstart/azure_edge_iot_ops/aks_edge_essentials_single_esa/_index.md

```diff
@@ -1,25 +1,21 @@
 ---
 type: docs
-title: "Discover Edge Storage Accelerator(Preview) on AKS Edge Essentials single node deployment"
-linkTitle: "Discover Edge Storage Accelerator(Preview) on AKS Edge Essentials single node deployment"
+title: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
+linkTitle: "Discover Edge Storage Accelerator on AKS Edge Essentials single node deployment"
```

Comment:
Discover should be removed.

The scenario should just be titled "Edge Storage Accelerator on AKS Edge Essential single node deployment"

Reply:
Discover is the start of the Akri scenario where you actually discover cameras with Akri so its just weird to see it reused here in a different way